### ISSUE TITLE
SVDquartets Column Sampling

### ISSUE THREAD

**Issue by j-chou** [j-chou/svdquartets#2]
I ran the whole pipeline on 5 of the phylip alignments and missing branch rates for both SVDQuartets+QMC and SVDQuartets+wQMC were always above 0.5 and for one alignment both rates were 1. I think this might a sampling issue: instead of taking a bunch of columns from the alignments, shouldn't SVDQuartets be taking one column from each alignment? (We have 1000 gene alignments per species tree).

**Comment by redavids** [comment]
Yes, please try it with just one column from each alignment first and see
what you get....

On Tue, Mar 31, 2015 at 10:44 AM, j-chou <notifications@github.com> wrote:

> I ran the whole pipeline on 5 of the phylip alignments and missing branch
> rates for both SVDQuartets+QMC and SVDQuartets+wQMC were always above 0.5
> and for one alignment both rates were 1. I think this might a sampling
> issue: instead of taking a bunch of columns from the alignments, shouldn't
> SVDQuartets be taking one column from each alignment? (We have 1000 gene
> alignments per species tree).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/j-chou/svdquartets/issues/2>.
>



-- 
Ruth Davidson
Department of Mathematics, University of Illinois Urbana-Champaign
http://www.math.uiuc.edu/~redavid2/
